app/package.json

```diff
@@ -6,7 +6,8 @@
     "preview": "vite build && vite preview",
     "pretty": "prettier --write \"**/*.css\" \"**/*.json\"",
     "lint": "eslint --fix \"**/*.js\" \"**/*.html\" && prettier --write \"**/*.js\" \"**/*.html\"",
-    "tailwind": "tailwindcss -i css/index.css -o css/prod.css"
+    "tailwind": "tailwindcss -i css/index.css -o css/prod.css",
+    "tailwind-watch": "tailwindcss -i css/index.css -o css/prod.css --watch"
```

Comment:
run this script in terminal to make tailwind rebuild prod.css when tw classnames are changed